### DISCUSSION TITLE
Nix

### DISCUSSION THREAD

**Discussion by samnystrom** [catppuccin/catppuccin#1992] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

A NUR repository.

### Any additional comments?

WIP port is at https://github.com/Stonks3141/ctp-nix.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closing as raised in #2103
